mne/epochs.py

```diff
@@ -3678,6 +3678,11 @@ def make_fixed_length_epochs(raw, duration=1., preload=False,
         Raw data to divide into segments.
     duration : float
         Duration of each epoch in seconds. Defaults to 1.
+    overlap : float
+        The overlap between epochs. Must be ``0 <= overlap < duration``.
+        Default is 0.
+
+        .. versionadded:: 0.23
```

Comment:
docstring entries must match the order of the parameters. So it should be moved just before verbose

Reply:
Thanks!